WebRoot/skins/_base/base4/skin.properties

```diff
@@ -2054,3 +2054,19 @@ ChooserListViewCustomStyleContainer     = @PanelBorder@; @FixBoxModel@;
 MetaPropertyTable       =
 MetaPropertyTableCell   = padding:@SkinWrapperPadding@ @WidgetInternalPadding@; padding-top:0;
 
+#######################################################################
+#   Task List view
+#######################################################################
+
+TaskListViewAddNewTaskIcon                                  = display:inline-block; margin-right:12px;
+TaskListViewNewTaskBannerWrapper                            = @Text-hint@ @BottomSeparator@ @InactiveCursor@ @FontSize-big@ padding:@SkinWrapperPadding@; padding-right:0px; padding-left:6px;
+TaskListViewHeaderItemLabel                                 = @FontSize-slightly-big@ @Text-light@ padding: 10px 0px 10px 11px;
+TaskReadingPaneRight-ListViewItemCheckboxCell               = padding-left:10px;
+TaskReadingPaneRight-ListViewItem                           = height:80px;
+TaskReadingPaneRight-ListViewItemSubject                    = vertical-align:bottom;
+TaskReadingPaneRight-ListViewItemDate                       = vertical-align:middle;
+TaskReadingPaneRight-ListViewItemTopRowContainer            = height:60%; margin:0;
+TaskReadingPaneRight-ListViewItemBottomRowContainer         = margin-top:-6px;
+TaskReadingPaneRight-ListViewItemBottomIconOuterWrapper     = padding-right:@SkinWrapperPadding@;
+TaskReadingPaneRight-ListViewItemBottomIconInnerWrapper     = margin-left:auto;
+TaskReadingPaneRight-ListViewItemCheckIcon                  = position:relative; top:38%;
```

Comment:
I think vertical align can also be used here as well and we can get rid of top with that.